bin/spec-configs/hpc2021/amdgpu_clang.cfg

```diff
@@ -96,22 +102,19 @@ PORTABILITY_LIBS = -lm
   532.sph_exa_t:
     MPIRUN_OPTS += --mca topo basic
 
+ 528.pot3d_t:
```

Comment:
remove these 3 lines, also point me at the system and location of this config where this appears.